frontend/src/libs/components/input/input.tsx

```diff
@@ -1,3 +1,4 @@
+import { clsx } from 'clsx';
```

Comment:
Let's import clsx only in one place (create smth like `getValidClassnames` helper and use it)

Reply:
fixed

Reply:
fixed